Tasks/NuGetInstallerV0/nugetinstaller.ts

```diff
@@ -188,7 +188,12 @@ async function main(): Promise<void> {
             credCleanup();
         }
 
+        if (!tl.getVariable('PASS_DEPRECATED_TASK')) {
```

Comment:
This would be safer: (but please test)

tl.getVariable('PASS_DEPRECATED_TASK').toLowerCase() === "true"

getVariable may not return the right variable type for this; ! on a string might not return the expected result

Reply:
In the latest changes this is now 

if(tl.getVariable('FAIL_DEPRECATED_TASK'))

Reply:
This is a bug.  The if() statement will always return false.

fix is to change to: 

tl.getVariable('PASS_DEPRECATED_TASK').toLowerCase() === "true"

Reply:
Changed this!

Reply:
Please use `if(tl.getBoolFeatureFlag('FAIL_DEPRECATED_TASK'))` instead.
https://github.com/microsoft/azure-pipelines-task-lib/blob/5ab4ebe0c1c1472e62e08fd10001983ed5d01e40/node/task.ts#L285